src/lib.rs

```diff
@@ -1,6 +1,189 @@
 use std::ffi::OsString;
 use std::iter::FromIterator;
 
+/// Structure that represent a unique Variable.
+#[derive(Clone, Debug)]
+pub struct Var {
+    /// The variable's key.
+    key: OsString,
+    /// The variable's value.
+    value: OsString,
+    /// Conditional statement for this variable.
+    condition: bool,
+    /// The environment that own the variable.
+    environment: Environment,
+}
+
+impl Var {
+    /// Check if the Var's value is strictly different of the input.
```

Comment:
`Check` implies that it returns a `bool`.

Trying to think of a way to more clearly convey that this makes the next command conditional.

`Conditionally apply the following action if ...`?